cluster/images/generate_imaging_script.sh

```diff
@@ -31,7 +31,7 @@ echo "#!$(which sh) -e"
 network="--network=${TYPE} --subnet=${TYPE}"
 [[ "${PROJECT}" = "${DEV_PROJECT}" ]] && network=""
 echo "$GCL instances create ${sourceInstance} --description=\"Instance for ${TYPE} disk image creation\" --zone=${ZONE} ${network} --boot-disk-size 100"
-echo "sleep 5"
+echo "sleep 20"
```

Comment:
unacceptable

Reply:
Oh dear. Have set to 10 seconds as a compromise. Haven't tested that it works for me but don't want to build a new image for no reason. Hopefully that is acceptable?